scripts/tests/test_zkai_d64_rmsnorm_swiglu_surface_probe.py

```
@@ -0,0 +1,307 @@
+            tmp = pathlib.Path(raw_tmp)
+            source = tmp / "arithmetic_subset_prover.rs"
+            source.write_text(
+                "\n".join(
+                    [
+                        "// validate_phase5_proven_fixture matches_linear_block_v4_with_lookup",
+                        "// matches_decoding_step_v2",
+                        'const SPOOF: &str = "validate_phase5_proven_fixture matches_linear_block_v4_with_lookup matches_decoding_step_v2";',
+                        'const MESSAGE: &str = "broader arithmetic-subset AIR coverage remains internal";',
+                        'const MESSAGE_2: &str = "experimental Phase12 carry-aware proving supports only the decoding_step_v2 family";',
+                    ]
+                ),
```

Ruff (0.15.12)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)